__tests__/errorsHandled.test.js

```diff
@@ -37,8 +37,68 @@ describe("GET /api/:invalid endpoint", ()=>{
         .get('/api/invalid')
         .expect(404) 
         .then((res)=>{
-            //console.log(Object.keys(res));
             expect(res.body.msg).toBe("Invalid endpoint");
         })
     })
+
+})
+
+describe("GET /api/reviews/:review_id", ()=>{
+    test("GET /api/reviews responds with a review object, which has the following keys: review_id, title, review_body, designer, review_img_url, votes, category, owner, create_at", ()=>{
+        return request(app)
+        .get('/api/reviews/1')
+        .expect(200)
+        .then((res)=>{
+            expect(res.body[0]).toEqual(expect.objectContaining({
```

Comment:
```js
expect(res.body[0])
```
You are asserting that you have an array with the review object in here. The body should be of the format 
```js
{
  review : {
     // review contents...
  }
}
```
Change this to expect the requested on object on a key of review instead of in an array.